python/res/enkf/site_config.py

```diff
@@ -77,22 +77,21 @@ def __init__(self, user_config_file=None, config_content=None, config_dict=None)
                     ext_job_list.add_job(job[ConfigKeys.NAME], new_job)
                 except:
                     print("WARNING: Unable to create job from {}".format(job[ConfigKeys.PATH]))
-                    continue
 
             for job_path in config_dict.get(ConfigKeys.INSTALL_JOB_DIRECTORY, []):
                 if not os.path.isdir(job_path):
-                    print("WARNING: Unable to locate job directory {}".format(job[ConfigKeys.PATH]))
+                    print("WARNING: Unable to locate job directory {}".format(job_path[ConfigKeys.PATH]))
```

Comment:
Shouldn't this be `"...".format(job_path)`?

Reply:
Not sure how I missed that, but yes it should be just `job_path`

I also missed the email notification for this PR and I just now noticed your comment when I remembered to check what was going on with this PR.